example/todeo/README.md

```diff
@@ -5,7 +5,7 @@ Todeo example project.
 ## Init commands
 
 ```sh
-fvm use 3.3.3
+fvm use 3.3.6
```

Comment:
There are newer, stable versions available. What do you think about using one of them?